sbc_translation/models/correspondence.py

```diff
@@ -112,36 +109,36 @@ def check_local_translation_done(self):
         tc = translate_connector.TranslateConnect()
         letters_to_update = tc.get_translated_letters()
 
-        if not letters_to_update == -1:
-            for letter in letters_to_update:
-                correspondence = self.browse(letter["letter_odoo_id"])
-
-                tg_lang = letter["target_lang"]
-                target_lang_id = self.env['res.lang.compassion'].search(
-                    [('code_iso', '=', tg_lang)]).id
-                # find the good text to writte
-                if tg_lang == 'eng':
-                    target_text = 'english_text'
-                elif tg_lang in correspondence.child_id.project_id\
-                        .country_id.spoken_lang_ids.mapped('code_iso'):
-                    target_text = 'translated_text'
-                else:
-                    raise AssertionError('The letter does not exist in odoo')
-
-                # UPDATE Odoo Database
-                correspondence.write(
-                    {target_text: letter["text"],
-                     'state': 'Received in the system',
-                     'destination_language_id': target_lang_id})
-
-                # Send to GMC
-                if correspondence.direction == 'Supporter To Beneficiary':
-                    pdb.set_trace()
-                    action_id = self.env.ref(
-                        'onramp_compassion.create_commkit').id
-                    self.env['gmc.message.pool'].create({
-                        'action_id': action_id,
-                        'object_id': letter["letter_odoo_id"]
-                    })
-
-                tc.remove_letter(letter["id"])
+        for letter in letters_to_update:
+            correspondence = self.browse(letter["letter_odoo_id"])
+
+            tg_lang = letter["target_lang"]
+            target_lang_id = self.env['res.lang.compassion'].search(
+                [('code_iso', '=', tg_lang)]).id
+            # find the good text to writte
+            if tg_lang == 'eng':
+                target_text = 'english_text'
+            elif tg_lang in correspondence.child_id.project_id\
+                    .country_id.spoken_lang_ids.mapped('code_iso'):
+                target_text = 'translated_text'
+            else:
+                raise AssertionError(
+                    'letter correspondence.kit_identifier was translated in\
```

Comment:
But after thinking, this field is not set before we send it to gmc so you can replace by `correspondence.id`